node/connection_base.js

```diff
@@ -266,6 +267,12 @@ TChannelConnectionBase.prototype.handleCallRequest = function handleCallRequest(
     }
 };
 
+TChannelConnectionBase.prototype.handlePingResponse =
```

Comment:
also once you get it there, it will be clear that the arg should be spelled `resFrame` to make it clearer